libs/designer/src/lib/ui/settings/settingsection.tsx

```diff
@@ -301,9 +303,11 @@ const Setting = ({ id, settings, isReadOnly }: { id?: string; settings: Settings
           return (
             <SettingTokenField
               {...{
+                useDisplaytextEditor: useDisplaytextEditor,
```

Comment:
are we able to up this a layer to parametersTab? We don't want to use node-specific logic on this layer